docs/tutorial.md

```diff
@@ -0,0 +1,285 @@
+# CytoSnake Tutorial
```

Comment:
Yes, this was autogenerated by using this package: https://rst-to-myst.readthedocs.io/en/latest/. I am not a fan of the line cutting either.

I do not know if this is a `myst` specific formatting

Reply:
@axiomcura - I wonder if `rst-to-myst` did something here to change the one-sentence-per-line structure (if it changed). I haven't used that tool enough to know whether it would change the content structure in that way. Generally, my experience with `myst` has been that it will correctly render one-sentence-per-line when the content is in place.